service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -136,8 +137,7 @@ private void sendNotificationForNewPlacementOrder(CallbackRequest callbackReques
         Placement currentPlacement = placementService.getPlacement(caseData, child);
         Placement previousPlacement = placementService.getPlacement(caseDataBefore, child);
 
-        // TODO refactor: this logic is confusing. Suggested: new method isNewOrUpdated... FPLA-1473
-        if (!isUpdatingExistingPlacement(previousPlacement, currentPlacement)) {
+        if (isNewPlacementApplication(previousPlacement, currentPlacement)) {
```

Comment:
the naming is still not quite right. We want to publish the event when a new OR an updated placement application. I think it is important to capture this